spec/index.bs

```diff
@@ -5907,6 +6054,58 @@ enum GPUBlendFactor {
 };
 </script>
 
+{{GPUBlendFactor}} defines how either a source or destination blend factors is calculated:
+
+<table class="data">
+  <thead>
+    <tr>
+      <th>GPUBlendFactor
+      <th>Blend factor RGBA components
+    </tr>
+  </thead>
+  <tbody dfn-type=enum-value dfn-for=GPUBlendFactor>
+    <tr>
+      <td><dfn>"zero"
+      <td><code>(0,0,0,0)
+    <tr>
+      <td><dfn>"one"
+      <td><code>(1,1,1,1)
+    <tr>
+      <td><dfn>"src"
+      <td><code>(R<sub>src</sub>,G<sub>src</sub>,B<sub>src</sub>,A<sub>src</sub>)
+    <tr>
+      <td><dfn>"one-minus-src"
+      <td><code>(1-R<sub>src</sub>,1-G<sub>src</sub>,1-B<sub>src</sub>,1-A<sub>src</sub>)
+    <tr>
+      <td><dfn>"src-alpha"
+      <td><code>(A<sub>src</sub>,A<sub>src</sub>,A<sub>src</sub>,A<sub>src</sub>)
+    <tr>
+      <td><dfn>"one-minus-src-alpha"
+      <td><code>(1-A<sub>src</sub>,1-A<sub>src</sub>,1-A<sub>src</sub>,1-A<sub>src</sub>)
+    <tr>
+      <td><dfn>"dst"
+      <td><code>(R<sub>dst</sub>,G<sub>dst</sub>,B<sub>dst</sub>,A<sub>dst</sub>)
+    <tr>
+      <td><dfn>"one-minus-dst"
+      <td><code>(1-R<sub>dst</sub>,1-G<sub>dst</sub>,1-B<sub>dst</sub>,1-A<sub>dst</sub>)
+    <tr>
+      <td><dfn>"dst-alpha"
+      <td><code>(A<sub>dst</sub>,A<sub>dst</sub>,A<sub>dst</sub>,A<sub>dst</sub>)
+    <tr>
+      <td><dfn>"one-minus-dst-alpha"
+      <td><code>(1-A<sub>dst</sub>,1-A<sub>dst</sub>,1-A<sub>dst</sub>,1-A<sub>dst</sub>)
+    <tr>
+      <td><dfn>"src-alpha-saturated"
+      <td><code>(A<sub>src</sub>,A<sub>src</sub>,A<sub>src</sub>,A<sub>src</sub>)
+    <tr>
+      <td><dfn>"constant"
+      <td><code>(R<sub>const</sub>,G<sub>const</sub>,B<sub>const</sub>,A<sub>const</sub>)
+    <tr>
+      <td><dfn>"one-minus-constant"
+      <td><code>(1-R<sub>const</sub>,1-G<sub>const</sub>,1-B<sub>const</sub>,A<sub>const</sub>)
+  </tbody>
+</table>
```

Comment:
nit: add spaces after commas and around minus signs, for readability?

Reply:
Done